<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>600</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Capture</string>
    </property>
    <widget class="QPushButton" name="registerImagesButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Register Images</string>
     </property>
    </widget>
    <widget class="QPushButton" name="runButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>921</width>
      <height>451</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>450</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>   Camera </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="cameraLabel">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>30</y>
       <width>891</width>
       <height>405</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>405</height>
      </size>
     </property>
     <property name="text">
      <string>Camera Screen</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>-1</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>921</width>
      <height>121</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Tool Timeline</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="addTool">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>19</y>
       <width>71</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Add Tool</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>811</width>
       <height>91</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>809</width>
        <height>89</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>   Program</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="SaveProgramButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>65</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoadProgramButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>65</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>25</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Program 
Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ProgramName">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>41</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>541</width>
      <height>80</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="Exit_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>65</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>100</y>
      <width>331</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>301</width>
       <height>381</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>299</width>
        <height>379</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="Report_Label">
         <property name="text">
          <string>### Reports Will Be Shown Here! ###</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="GuideLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Status:
 Steady</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="RunMode">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>30</y>
      <width>91</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>Go to 
Run Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
